docs/prepping-local-system-runbook-windows.md

```diff
@@ -254,7 +254,7 @@ For the `platform/installer` image, the :tag represents the version of Pega you
 For `platform/pega` and `platform/search` images, Pega also offers an image with a version tag appended with a datestamp using the pattern `pegaVersion-YYYYMMDD` to indicate the version and the date that Pega built the image. For example, if you pull the `platform/pega` with a tag, `pega:8.5.1-20201026`, the tag indicates that Pega built this 8.5.1 image on 26 October 2020. Using the version tag without the datestamp will always point to the most recently built image for that version.
 
 The Docker image `platform-services/search-n-reporting-service` is used with the [Search and Reporting Service](../charts/backingservices/charts/srs) under [backing services](../charts/backingservices). The SRS subchart deployment provisions the latest generation of search and reporting capabilities in Pega Infinity. This service is optional and is an alternate search feature to the previously used `platform/search` image-based Elasticsearch deployment. To configure this service, see [Instructions to configure the Pega backingservices](../charts/backingservices/README.md).
-The docker image format is different for backing services and does not follow the datestamp pattern. For installations of or upgrades to Pega Infinity 8.6, you may configure Docker image `platform-services/search-n-reporting-service` with tag `<srs-version>` compatible with the Pega version as provided in [compatibility matrix](../charts/backingservices/README.md#srs-version-compatibility-matrix).
+The docker image format is different for backing services and does not follow the datestamp pattern. For installations of or upgrades to Pega Infinity 8.6, you may configure Docker image `platform-services/search-n-reporting-service` with tag `<srs-version>` compatible with the Pega version as provided in [compatibility matrix](../charts/backingservices/charts/srs/README.md#srs-version-compatibility-matrix).
```

Comment:
@khick77 should confirm that the docker image details should be removed from the linux and windows prepping runbooks. We moved all of these details in August 2022 to>https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images

I can update the existing wording there to use this new link>>
The docker image format is different for backing services and does not follow the datestamp pattern. For installations of or upgrades to Pega Infinity 8.6, you may configure Docker image `platform-services/search-n-reporting-service` with `<srs-version>` compatible with the Pega version as provided in [compatibility matrix](../charts/backingservices/charts/srs/README.md#srs-version-compatibility-matrix).

Reply:
@taz-pega-work 
My PR is here:
https://github.com/pegasystems/pega-helm-charts/pull/389/files
it was merged, and only affected the readme.md file.  Dave's changes to that file seem to be putting back in the notes that I had ripped out.  I'll leave it to you to decide if that's what you want or not.  It sounds like not.

Reply:
we don't want the information to live in two places. Clients need to use this information and I think the goal of your PR was to also take out the detailed Docker image information out of each prepping runbook and the readme and the two runbooks would then just point to the latest information. For 8.7 that's >>https://docs.pega.com/client-managed-cloud/87/pega-provided-docker-images
and for 8.8, it will be here for 8.8 GA>https://docs.pega.com/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html
since that won't be published until mid-October, you can see the content here until then>>https://pega-prod.zoominsoftware.io/bundle/platform-88/page/platform/deployment/client-managed-cloud/pega-docker-images-manage.html
I think for the 8.8 timeframe, we need to have Dave's changes go into 8.8 version, since right now, that directory structure refers to won't be live until around 8.8 GA. 

Does this make sense??